### PR TITLE
1009 community logo

### PR DESCRIPTION
**Pull request by criedlberger**

### CODE REVIEW

File: app/controllers/concerns/hydranorth/communities_controller_behavior.rb [hunk 1]
```diff
@@ -12,5 +12,12 @@ module CommunitiesControllerBehavior
 
     def index
     end
+
+    def logo
+      @community = Collection.find(params[:id])
+      logger.debug @community.logo
```
Comment (weiweishi) [inline app/controllers/concerns/hydranorth/communities_controller_behavior.rb]: debug code should be removed.

File: app/views/collections/show.html.erb [hunk 2]
```diff
@@ -11,12 +11,9 @@
           <%= sanitize @presenter.description %>
         </div>
       </header>
-      <% unless has_collection_search_parameters? %>
-      <%= render 'collections/show_descriptions' %>
-      <% end %>
     </div>
     <div class="col-sm-2">
-      <%= render partial: 'collections/media_display', locals: { generic_file: @collection } %> 
+      <%= image_tag url_for(:controller => 'communities', action: 'logo', id: @collection.id), alt: "No logo" %>
```
Comment (weiweishi) [inline app/views/collections/show.html.erb]: It seems that there's no control over the size of the image allowed on the upload/display. I uploaded a medium size logo file for testing, and it overlapped with the community title. We will probably need both some css work to control the allowed size.

File: app/models/datastreams/logo_datastream.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class LogoDatastream < ActiveFedora::File
+  include Sufia::FileContent::Versions
```
Comment (weiweishi) [inline app/models/datastreams/logo_datastream.rb]: We may need to have some validations in the model to validate file format, size etc?

**Comment by pgwillia** [comment]
@criedlberger @weiweishi is this ready to go?  Should we still be merging with Sprint39?

**Comment by weiweishi** [comment]
@pgwillia @criedlberger I think we should try to merge it with current Sprint branch. The debugging code has not yet been removed but I think Chris has addressed the issue with the image size checking. I think it's ready to be merged.

**Comment by criedlberger** [comment]
@pgwillia, @weiweishi I have created a new pull request for the current sprint

**Pull request closed without merging**